content/glossary.md

```diff
@@ -3,19 +3,52 @@ title: Glossary
+
+See the [API documentation][1] for more information.
+
 **Check**
 
+Checks are small Python programs run periodically by the Agent. A Check performs an action and then gathers the result, which the Agent then stores and reports to the Datadog platform. These programs are freeform and are generally used to collect metrics from custom environments or applications.
+
+Note that the word “check” — when not capitalized — refers to the generic act of taking a measurement.
+
+**Client Library**
+
+There are a number of libraries in a variety of languages in order to help you instrument your applications with Datadog directly. Datadog provides official libraries for C#, Golang, Java, PHP, Python, and Ruby. Libraries in other languages have been contributed by the community, and are supported on a best-effort basis.
+
+Libraries are designed to interact with either the API or DogStatsD, though a handful provide support for both.
+
+See the [Libraries documentation][2]
+
 **Dashboard**
 
+**DogStatsD**
+
+DogStatsD refers to two related things: a protocol based on [StatsD][3], and an application for reporting metrics which implements that protocol. The DogStatsD protocol is an extension of the StatsD protocol, with some modifications that are specific to the Datadog platform. The DogStatsD application is a service that is bundled with the Agent, and is used as a lightweight mechanism for reporting metrics.
+
+See the [DogStatsD documentation][4] for more information.
+
 **Downtime**
 
 **Embed**
 
+**Integration**
+
+An Integration is a way to get data from your systems into Datadog. Integrations gather data from a given source, ensure that those data are classified correctly, and provide some other assets to assist with configuration and usage. Data sources can be pretty much anything from daemons on a server, to cloud services, to third-party APIs, and more.
+
+See the [Integrations documentation][5] for more information.
+
 **Monitor**
 
-**Multiple-organization accounts (multi-org accounts)**. Multi-org accounts allow a parent-organization to create multiple child-organizations. This is most often used with Managed Service Providers (MSPs) that have many customers. Each customer requires their own organization that is not accessible by other customers. All billing for multi-org accounts will be performed on the parent-organization and usage reports for the parent-organization will include usage information for all child-organizations.
+**Multiple-organization accounts (multi-org accounts)**
+
+Multi-org accounts allow a parent-organization to create multiple child-organizations. This is most often used with Managed Service Providers (MSPs) that have many customers. Each customer requires their own organization that is not accessible by other customers. All billing for multi-org accounts will be performed on the parent-organization and usage reports for the parent-organization will include usage information for all child-organizations.
```

Comment:
should parent-org... and child-org... have the hyphen?

Reply:
Idk - this was in the original file. I just added carriage returns.